l10nfr.properties

```diff
@@ -1,14 +1,20 @@
 category=Localization
 description=Language Pack for French
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-fr
-publicVersions=1.15.1
+publicVersions=1.15.1,9.8.0
 archivedVersions=
 
-defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
+defaults.mavenGroupId=org.sonarqube.l10n.fr
 defaults.mavenArtifactId=sonar-l10n-fr-plugin
 
 1.15.1.description=Support SonarQube 6.7.3+
-1.15.1.sqVersions=[8.9,LATEST]
+1.15.1.sqVersions=[8.9,9.7.*]
```

Comment:
```suggestion
1.15.1.sqVersions=[8.9,9.7.*]
1.15.1.mavenGroupId=org.codehaus.sonar-plugins.l10n
```

Reply:
I think we're going to need this because you're changing the default GroupId, and this version doesn't have the new GroupId. Could you add this please, before I approve?